util/stmtsummary/statement_summary.go

```diff
@@ -81,6 +81,10 @@ type stmtSummaryByDigestMap struct {
 		sessionEnabled string
 		// setInSession indicates whether statement summary has been set in any session.
 		globalEnabled string
+		// sessionInternalQueryEnabled indicates whether internal statement summary is enabled in current server.
+		sessionInternalQueryEnabled string
+		// globalInternalQueryEnabled indicates whether internal statement summary has been set in any session.
+		globalInternalQueryEnabled string
```

Comment:
How about only keep a global variable here, I think the session variable is a little redundant. @djshow832

Reply:
I'm going to refactor it. Keep it for now.

Reply:
It's strange that other variables support both global and session scope, but this one only supports global scope.